sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -0,0 +1,514 @@
+namespace Azure
+{
+    public abstract partial class AsyncPageable<T> : System.Collections.Generic.IAsyncEnumerable<T> where T : notnull
```

Comment:
Hmm what is this all about?

Reply:
This file got produced when I ran Export-API.ps1

Reply:
I think it is fine to leave it off assuming CI still passes without it, since it is not related to the changes in this PR.